<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="main">
     <widget class="QPushButton" name="goToSettings">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Настройки</string>
      </property>
     </widget>
     <widget class="QComboBox" name="group">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Преподаватель</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Генерировать</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Академическая группа</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Генератор</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <widget class="QPushButton" name="goBackButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">&lt; Назад</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Журнал ошибок</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Сохранять события в</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>156</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Адрес api</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="apiPath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>156</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Настройки</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshData">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Обновить данные</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="eventDir">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chooseEventDir">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="journalDir">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chooseJournalDir">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="saveSettings">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuICS_gen">
    <property name="title">
     <string>ICS-gen</string>
    </property>
   </widget>
   <addaction name="menuICS_gen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
